### PR TITLE
More new studio page compatibility

### PR DESCRIPTION
**Pull request by mxmou**
### Changes

Changes to `exact-count`, `dark-www` and `scratchr2` for the scratch-www studio page.

### Screenshots

image
image

### CODE REVIEW

**Comment by apple502j** [comment]
dark-www and scratchr2: enhancement
exact-count: bug (exact count is added after capped count, this occurs because capped count is a new addition)

**Comment by mxmou** [comment]
@apple502j This is correct, unless you consider showing dark icons on a dark background a bug.

File: addons/exact-count/studio.js [hunk 1]
```diff
@@ -21,16 +21,17 @@ export default async function ({ addon }) {
...
-    countProjects(apiUrlPrefix, 0, 100, function (count) {
-      if (addon.tab.clientVersion === "scratch-www")
-        document.querySelector(".studio-tab-nav span").innerText += ` (${count})`;
-      else document.querySelector("[data-count=projects]").innerText === count;
+  const apiUrlPrefix =
+    "https://api.scratch.mit.edu/studios/" + /[0-9]+/.exec(location.pathname)[0] + "/projects/?limit=40&offset=";
+  const initialDelta = 100;
+  if (addon.tab.clientVersion === "scratch-www") {
+    const countElement = await addon.tab.waitForElement(".studio-tab-nav > a:first-child .tab-count");
+    countProjects(apiUrlPrefix, 0, initialDelta, async function (count) {
```
Comment (apple502j) [inline addons/exact-count/studio.js:29]: This isn't async right?

**Comment by WorldLanguages** [comment]
> exact-count: bug (exact count is added after capped count, this occurs because capped count is a new addition)

Technically a bug but probably not enough for v1.15.1
Release is planned for June ~15th which is "mid-June", so we might release before or after studio page update, we don't know yet

**Comment by mxmou** [comment]
The release date for new studios has been announced on Transifex:

image

**Pull request merged**